libs/gi/db/src/Database/DataManagers/OptConfigDataManager.ts

```diff
@@ -85,6 +85,8 @@ export interface OptConfig {
   compareBuild: boolean
   levelLow: number
   levelHigh: number
+  useLoadoutExclusion: boolean // teammates loadout exclusion flag
+  loadoutExclusion: string[]
```

Comment:
I don't think its a good idea to store this value in the database, because this value is calculatable.